examples/plot_pp_study.py

```diff
@@ -0,0 +1,362 @@
+
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Predictive Policing Case Study
+================================
+"""
+# %%
+#
+#
+# Contents
+# ========
+#
+# 1. Overview
+# 2. Background
+# 3. Fairlearn Results
+# 4. Problem area & Sociotechnical context
+# 5. Conclusion
+#
+# Overview
+# --------
+#
+# In this document we outline a sample sociotechnical case study - using a
+# hypothetical predictive policing scenario - based on data found in the
+# `Boston housing dataset <https://www.kaggle.com/c/boston-housing>`_
```

Comment:
There's quite a bit of a jump from "we're using a housing dataset" to "we predict crime rate". As a reader I'm confused by this. How does that connection make sense? Does it at all? I think it's worth pointing out how such a design choice can affect the outcome.

Reply:
Exactly. I think either way is fine for demonstrating that it's not good, we just want to be very clear :-)